habitat/plan.sh

```diff
@@ -1,68 +1,76 @@
 pkg_name=inspec
 pkg_origin=chef
-pkg_version=$(cat "VERSION")
 pkg_description="InSpec is an open-source testing framework for infrastructure
   with a human- and machine-readable language for specifying compliance,
   security and policy requirements."
-pkg_upstream_url=https://github.com/chef/inspec
+pkg_upstream_url=https://www.inspec.io/
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('Apache-2.0')
-pkg_source=false
 pkg_deps=(
+  core/busybox-static
+  core/cacerts
   core/coreutils
   core/libxml2
   core/libxslt
-  core/ruby
   core/net-tools
-  core/busybox-static
+  core/ruby
 )
 pkg_build_deps=(
   core/bundler
-  core/coreutils
   core/gcc
   core/make
   core/readline
+  core/sed
 )
 pkg_bin_dirs=(bin)
 
-do_download() {
-  return 0
+pkg_version() {
+  cat VERSION
 }
 
-do_verify() {
-  return 0
-}
-
-do_unpack() {
-  return 0
+do_before() {
+  if [[ ! -f VERSION ]]; then
+    exit_with "habitat/VERSION not found. Run 'rake release_habitat' to ensure it exists" 1
+  fi
+  update_pkg_version
 }
 
 do_prepare() {
   # Create a Gemfile with what we need
-  cat > Gemfile <<GEMFILE
+  cat > "$CACHE_PATH/Gemfile" <<GEMFILE
```

Comment:
I think that's intended currently. Ultimately I'd rather this be part of a CI pipeline that cut a release anytime we promoted to rubygems, but seeing as how these tasks can sometimes be async, this feels like the safest way for now.